src/layout/Footer.jsx

```diff
@@ -1,7 +1,95 @@
 import React from "react";
+import { Link as GatsbyLink } from "gatsby";
+import { makeStyles } from "@material-ui/core/styles";
+import Typography from "@material-ui/core/Typography";
+import Link from "@material-ui/core/Link";
+import { Facebook, Twitter, RssFeed } from "@material-ui/icons";
+import Grid from "@material-ui/core/Grid";
+
+/* Defines CSS classes to use for all instances of this component. This is
```

Comment:
No need for this comment

Reply:
What if I forget how to use material ui

Reply:
Looks weird when we have the same logic in other places already without comments imho

Reply:
Ok, we'll make an actual documentation about this pattern in the docs folder